docs/welcome.mdx

```diff
@@ -38,140 +38,94 @@ improved hourly.
 - Permission and Role Management.
 - Delegation via OAuth2 and OpenID Connect.
 - Zero Trust Networking.
 - Modern API design with partial support for gRPC.
 
 ## The Ory Network
 
-The Ory Network is the commercial offering of Ory and is built on top of our popular open-source software. Our goal with The Ory
-Network is to offer a planet-scale, low-latency, resilient, and secure service that's easy to use and set up.
+The Ory Network is the commercial offering of Ory and is built on top of Ory Open Source software. The goal with Ory Network is to
+offer a planet-scale, low-latency, resilient, and secure service that's easy to use and set up.
 
-In short: The Ory Network is the most convenient and straightforward way to run Ory. Sign up at
-[console.ory.sh](https://console.ory.sh/registration) for an account and create a free developer project.
+In short: Ory Network is the most convenient way to run Ory. [Sign up](https://console.ory.sh/registration) and create a free
+developer project.
 
-### Components
+## Components
 
-The Ory Network uses a variety of components - some are based on open-source projects and others are closed source.
+Each project in Ory Network is an isolated tenant and uses many components providing functionality, user interfaces, and APIs
+around identities, sessions, login, OAuth2, permissions, and more. The core components of projects in Ory Network are
+[Ory Open Source servers](https://github.com/ory/).
 
-### Ory Console
-
-Ory Console is the management UI of The Ory Network. It offers capabilities such as showing and searching for identities, managing
-user sessions, and gaining valuable insights in the form of dashboards and customization.
-
-### Ory Account Experience
-
-Ory Account Experience implements screens such as login, registration, account recovery, account setting, and account
-verification. This allows for fast adoption of Ory.
-
-Contrary to other vendors, Ory allows you to implement your own (login, registration, ...) UI by offering simple, headless APIs.
-
-### Identity & session services
+### Identities and sessions
 
-The Ory Network incorporates the open-source [Ory Kratos Identity Server](https://www.ory.sh/kratos) project and offers:
+Ory Network incorporates the open-source [Ory Kratos Identity Server](https://www.ory.sh/kratos) and offers:
 
 - **Self-service** flows are everything users do on their own / without the help of others:
-- Registration with passwords, social sign in, OpenID Connect, ...
-- Login with passwords, social sign in, OpenID Connect, ...
-- Updating the profile, email, changing the password, un/linking with Social. Sign In providers, ...
+- Registration with passwords, social sign-in, OpenID Connect, ...
+- Login with passwords, social sign-in, OpenID Connect, ...
+- Updating the profile, and email, changing the password, un/linking with Social. Sign In providers, ...
```

Comment:
```suggestion
- Updating the profile, email, changing the password, un/linking with social sign-in providers, ...
```

Reply:
I can see that Grammarly made you add an "and" here but trust me - it doesn't make sense.